packages/core/src/families/solana/serializer.ts

```diff
@@ -0,0 +1,26 @@
+import BigNumber from "bignumber.js";
+import type { SolanaTransaction, RawSolanaTransaction } from "./types";
+
+export function serializeSolanaTransaction({
+  family,
+  amount,
+  recipient,
+}: SolanaTransaction): RawSolanaTransaction {
+  return {
+    family,
+    amount: amount.toString(),
+    recipient,
```

Comment:
model is required. https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledger-live-common/src/families/solana/transaction.ts#L34